search/scorer/scorer_term.go

```diff
@@ -58,6 +59,30 @@ func NewTermQueryScorer(queryTerm []byte, queryField string, queryBoost float64,
 	return &rv
 }
 
+func NewFuzzyTermQueryScorer(queryTerm []byte, queryField string, queryBoost float64,
```

Comment:
Can we reduce code here by having the new constructor delegate to the old one, and then we just toggle the scoreFuzzy field before returning it.  Also, why do we include fuzzyQuery as an option, presumably one would only use this constructor to create fuzzy queries.  I'm thinking something like:

func NewFuzzyTermQueryScorer(a,b,c) *TermQueryScorer {
  rv := NewTermQueryScorer(a,b,c)
  rv.scoreFuzzy = true
  return rv
}

Reply:
whoops..I should have done it this way..